app/broadcast/server/config.js

```diff
@@ -0,0 +1,15 @@
+import { Meteor } from 'meteor/meteor';
+import { settings } from '../../settings';
+
+Meteor.startup(() => {
+    settings.addGroup('Broadcast', function() {
+		// the channel for which discussions are created if none is explicitly chosen
```

Comment:
@bhardwajaditya I see you have copied the whole file `app/discussion/server/config.js` here, would you please remove this comment, it's not relevant here.